config.yaml

```diff
@@ -305,3 +305,19 @@ options:
       .
       Changing this config option will restart openvswitch-switch, resulting
       in an expected data plane outage while the service restarts.
+  enable-version-pinning:
+    type: boolean
+    default: true
```

Comment:
@sombrafam Could you comment on the default here? Commit f68ecbd10d2109014c67100ec7e7fe8e1823a7ec uses `false` here.

Reply:
"Having the pinning enabled by default causes unnecessary grief for
anyone already upgraded to OVN 22.03, so it's time to turn it off
by default."

Since we are backprting this to 20.03, the default should be true since it was causing problems only on 22.03.

Reply:
Ok, that makes sense. Thanks!